{
  "contest": 415,
  "handle": "4B",
  "risk": "2",
  "title": "Incorrect function computation in `PoolV3::_amountWithWithdrawalFee`",
  "issueId": 358,
  "issueUrl": "https://github.com/code-423n4/2024-07-loopfi-findings/issues/358",
  "originalIssueId": 538,
  "originalIssueUrl": "https://github.com/code-423n4/2024-07-loopfi-validation/issues/538"
}